packages/fuel-indexer-tests/scripts/test-examples.bash

```diff
@@ -6,61 +6,83 @@
+
+  if [[ "$db_status" == "$client_status" ]] && [[ "$db_status" == '"OK"' ]]; then
+    echo "Database and client are up and running."
+  else
+    echo "Database and/or client are not up and running."
+    exit 1
+  fi
+}
+
+export PGPASSWORD=my-secret
+
 set -ex
 
+forc_index_kill() {
+  # Shutdown the service
+  forc-index kill
+  sleep 2
+}
+
 # ********************************
-# examples/hello-indexer
+# examples/greetings
 # ********************************
 
-# Go to example
-cd examples/hello-world/
+cd examples/greetings/
 
-# Start the local Fuel node in the background
-cargo build -p hello-world-node
-cargo run -p hello-world-node --bin hello-world-node &
-sleep 2
+cargo build -p greetings-fuel-client
+cargo run -p greetings-fuel-client --bin greetings-fuel-client &
+sleep 1
 
-# Start service in the background
 forc-index start --postgres-password my-secret --run-migrations &
-sleep 2
-
-# Deploy the example indexer
-forc-index deploy --path ./hello-indexer
-sleep 5
-
-# Ensure service is up and running via health check
-db_status=$(curl http://localhost:29987/api/health | json_pp | jq '.database_status')
 sleep 1
-client_status=$(curl http://localhost:29987/api/health | json_pp | jq '.client_status')
 
-if [[ "$db_status" == "$client_status" ]] && [[ "$db_status" == '"OK"' ]]; then
-  echo "Database and client are up and running."
-else
-  echo "Database and/or client are not up and running."
-  exit 1
-fi
+forc-index deploy --path ./greetings-indexer
+sleep 3
 
+# check_service_status
 
 # Trigger an indexable event
-cargo run -p hello-world-data --bin hello-world-data
-sleep 2
+cargo run -p greetings-data --bin greetings-data
+sleep 1
 
 # Check data is indexed
-export PGPASSWORD=my-secret
-echo "SELECT COUNT(*) FROM fuel_examples.salutation;" | psql -h localhost -p 5432 -U postgres
+echo "SELECT COUNT(*) FROM fuellabs_greetings.salutation;" | psql -h localhost -p 5432 -U postgres
 sleep 2
 
-# Shutdown the service
-forc-index kill
-sleep 2
+forc_index_kill
 
 # ********************************
-# examples/hello-indexer-native
+# examples/greetings-native
 # ********************************
 
-# TODO: https://github.com/FuelLabs/fuel-indexer/issues/1069
+cd ../greetings-native
+
+forc-index build --path ./greetings-native-indexer --native
+
+# Update manifest with paths relative to the example due to cargo workspace
+sed -i "s|^graphql_schema: examples/greetings-native/greetings-native-indexer/schema/greetings_native_indexer\.schema\.graphql|graphql_schema: $PWD/greetings-native-indexer/schema/greetings_native_indexer.schema.graphql|" ./greetings-native-indexer/greetings_native_indexer.manifest.yaml
+
+# Start indexer in the background
+forc-index run-native --path ./greetings-native-indexer -- --run-migrations --postgres-password my-secret
+sleep 60
 
-cargo build -p hello_indexer_native --locked --release
+check_service_status
+
+# cargo run -p greetings-data --bin greetings-data
```

Comment:
- There's something odd happening on CI here where starting this indexer sort've blocks the rest of CI - even if the command is launched in the background. 
- I want to continue to look into this but don't want to block the PR
- You can confirm that running these steps locally works